src/coreComponents/meshUtilities/ComputationalGeometry.cpp

```diff
@@ -52,21 +52,16 @@ R1Tensor LinePlaneIntersection( R1Tensor lineDir,
 }
 
 //*************************************************************************************************
-real64 ComputeSurfaceArea( array1d< R1Tensor > const & points,
-                           localIndex const numPoints,
-                           R1Tensor const & normal )
+real64 ComputeSurfaceArea( array1d< R1Tensor > const & points )
 {
-  // reorder points counterclockwise
-  array1d< R1Tensor > pointsReordered = orderPointsCCW( points, numPoints, normal );
```

Comment:
I would think this might break something?

Reply:
It looks like I am the only one using this function so nothing has been broken. The PR passes the tests apart from the diffs coming from some new fields. I will check again though.